docs/development/pigs/pig-006.md

```diff
@@ -0,0 +1,114 @@
+
+# PIG 6 - Data/GTI handling in Gammapy
+
+* Author: David Fidalgo, Christoph Deil, Régis Terrier et al
+* Created: October 10, 2018
+* Accepted: --
+* Status: Being discussed
+* Discussion: --
+
+## Abstract
+
+In this PIG we want to outline the improvement of the IACT DL3 data handling in Gammapy.
+The current handling has some limitations and misses some key features, like selecting a given time interval and performing an analysis on it.
+Since specifics of the final data format of CTA are still unclear, this PIG does not aim for a final implementation but tries to solve the current limitations.
+This PIG emerged at the Coding Spring in Madrid 2018.
+
+## Some terminology
+
+This terminology and the definition of the terms will probably change or be expanded once the final data format of CTA is decided.
+For now we will use following definitions to go forward in Gammapy:
+
+* **GTI (good time interval)**:
+  * time intervals for which the instrument was on
+  * the sum over these time intervals equals the *on time*
+
+
+* **Observation**:
+  * has a unique identifier, the *obs_id*
+  * contains 1 event list, 1 GTI table and 1 corresponding set of IRFs (aeff, edisp, psf)
+  * has 1 dead-time fraction (`DEADC`)
+  * the live time of the observation is obtained as `ONTIME * (1 - DEADC)`
```

Comment:
Can you try to flesh out the GTI part of the PIG a bit?
You know, be more clear what we assume about GTI and how we use it, to have a more focused discussion.

I think we can make GTI required and fully rely on it being there.
Maybe, if easy, we could allow loading data without a GTI if TSTART and TSTOP is given in the EVENTS header, but then IMO the first step should be to create a GTI table, not try to drag that alternative format to have run times along in Gammapy.

Why have GTI at all and not just START and STOP for an observation?
No strong reason, it's just a little more flexible, e.g. if there's 5 seconds of hardware problems or a meteor, it can be cut out without having to start a new EVENTS and observation, if some IACT wants that.
It's also closer then to the data model that Fermi-LAT uses already, they absolutely require GTI.

My thinking for DEADC is that it should go away:
https://github.com/open-gamma-ray-astro/gamma-astro-data-formats/issues/62#issuecomment-428221596
For now, we should keep it, because some people might use it.
But I'd suggest linking to that issue from the PIG and trying to design it as much as possible that DEADC is an implementation detail that can be removed later without having to touch high-level code.

For PHASE, you could also try to be a bit more specific. For me, the key point in this PIG is to solve the use case of GTI and PHASE, and to defer other things like EVENT_TYPE to a future PIG. Suggest you state that explicitly, that this will not be solved here, but next year in a new PIG. But for PHASE, I think it's easy to solve in a generic solution like for GTI: add a table with PHASE START / STOP intervals, to allow a user to run an analysis with any PHASE selection they like. Do you think it's a good idea, or do you want to not develop the full solution how to do PHASE in this PIG?

Final comment: I'm not sure having the PIG in markdown is the best solution. RST is almost the same, and there shouldn't be duplication of content in one document and another notebook to avoid confusion / extra work during the review and discussion in the coming days (weeks?). Also version history will be lost when you rename MD to RST. So suggest to change to RST now. But it's not a big point, if you really want to do the review / discussion with MD and change later, OK.